Us 01 Signup API Implementation

FlorenzoC opened the pull request:
Fets canvis i merge

FlorenzoC committed:
Nova versió

He posat que a signup.py es intenti crear usuari, i si peta saltara el try-except de signup i retornara error, al David ja li compila

FlorenzoC committed:
Error parentesis

3nr1c reviewed (changes requested):
Falta el hashing de la password

ifrins commented on backend/app/faceduck/core/signup.py:
```diff
@@ -0,0 +1,8 @@
+import uuid
+
+def createUser(user, email, password, name, surname, birthday, gender):
+		newUser = User({meta='id': uuid.uuid4()},  username=user, email=email, password=password, name=name, surname=surname, birthday=birthday, gender=gender)
```
La validació de unicitat i tot plegat?

FlorenzoC committed:
Implementat SignUp + password hash

A mi em compila tot, crec que hauria de funcionar, hi podeu donar un cop d'ull?

IvanSevilla reviewed (approved):
Lo veo correcto

3nr1c reviewed (approved):
## Abans de fer merge, prova-ho. Si no saps com fer-ho, fes merge i s'encarregarà algú altre de fer el test (veure slack)

3nr1c commented:
# Tanco per canvi a una altra PR (mala praxis)
